src/entryPoints/AuthModule/UserInfo/UserInfoTypes.res

```diff
@@ -0,0 +1,10 @@
+type userInfo = {
+  email: string,
+  is_two_factor_auth_setup: bool,
```

Comment:
can we define in Js way like camel casing the variable like every type in this as JS common is camel case not snake case